### DISCUSSION TITLE
Umm... still a bug...

### DISCUSSION THREAD

**Discussion by MrPRAINGLE** [discordjs/discord.js#6942] [Q&A] [answered]
Hey, I dunno what did I do wrong, but the "blabla${data}" technique don't work, and that since... the beginning, in fact. What would I do to fix this ?

Thanks

**Comment by monbrey** [comment, marked as answer]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

I struggle to see how this is related to discord.js - please keep the GitHub Discussions on topic. This is not a general JavaScript support forum.

**Reply by MrPRAINGLE** [reply]
Ok, sorry... Anyway, thanks, it helped me 😁
